style.css

```diff
@@ -0,0 +1,1422 @@
+:root {
+  --primary-color:#FFFFFF;
+  --secondary-color: #0D2843;
+  --button-color-text:#5297FA;
+  --color-block-2:#E5E5E5;
+  --color-small-text: #0D2843;
+  --color-small-blue-cirle: #31D4DE;
+  --color-small-dark-blue-circle: #529AFE;
+  --color-small-purple-circle: #8959EF;
+  --color-background:linear-gradient(180deg, #4B95FA 0%, #CD4BFA 100%);
+  --black-color: #000000;
```

Comment:
we name colors with their names, not where we use it

and you don't need to write color


for finding names use a tool I have given to you

Reply:
Fixed